R/python.R

```diff
@@ -981,6 +981,15 @@ py_str.python.builtin.object <- function(object, ...) {
   str
 }
 
+#' @export
+py_str.python.builtin.bytearray <- function(object, ...) {
+  builtins <- import_builtins()
+  paste(
+    paste0("python.builtin.bytearray (length ", builtins$len(object), "):"),
+    capture.output(str(py_to_r(ba), give.length = FALSE, give.head = FALSE, vec.len = 10))
```

Comment:
Looks like that `ba` should be `object`?

Reply:
Good catch! I actually just changed this to just print the number of bytes and the type (as the initial implementation actually converted the bytearray to RAW which is probably too expensive for a print method)